match.py

```diff
@@ -87,7 +91,12 @@ def do_match(c):
                     )
                 else:
                     project.upload_clks(system, c.get_clks_raw(system, project_name))
-            project.start_run(c.matching_threshold)
+            # should we consider making the threshold a property of the project itself?
```

Comment:
If you mean change the "projects" key in config from a list of project names, to a dict where the project names are the keys and project-specific settings are in the values, then that makes sense. 
I'm not sure we have any other project-specific settings other than threshold though, so let's save that for whenever we find our second instance, just to minimize changes.

Reply:
What I mean by that is that it seems more streamlined (in my head at least) to tuck the threshold definition in with other variables during the instantiation of the Project object on [line 42](https://github.com/mitre/linkage-agent-tools/blob/multi-threshold/match.py?plain=1#L42)

Reply:
but ultimately I agree, let's save that for later.